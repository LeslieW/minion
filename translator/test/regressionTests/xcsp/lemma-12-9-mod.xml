<?xml version="1.0" encoding="UTF-8"?>

<instance>
<presentation name="?" maxConstraintArity="3" format="XCSP 2.0"/>

<domains nbDomains="1">
<domain name="D0" nbValues="9">0..8</domain>
</domains>

<variables nbVariables="12">
<variable name="V0" domain="D0"/>
<variable name="V1" domain="D0"/>
<variable name="V2" domain="D0"/>
<variable name="V3" domain="D0"/>
<variable name="V4" domain="D0"/>
<variable name="V5" domain="D0"/>
<variable name="V6" domain="D0"/>
<variable name="V7" domain="D0"/>
<variable name="V8" domain="D0"/>
<variable name="V9" domain="D0"/>
<variable name="V10" domain="D0"/>
<variable name="V11" domain="D0"/>
</variables>

<predicates nbPredicates="1">
<predicate name="P0">
  <parameters>int X0 int X1 int X2 int X3 int X4 int X5</parameters>
  <expression> 
    <functional>and(ne(X0,X1),and(ne(X2,X3),ne(X4,X5)))</functional>
  </expression>
</predicate>
</predicates>

<constraints nbConstraints="30">
<constraint name="C0" arity="3" scope="V0 V1 V2" reference="P0">
  <parameters>V0 V1 V0 V2 V1 V2</parameters>
</constraint>
<constraint name="C1" arity="3" scope="V0 V2 V3" reference="P0">
  <parameters>V0 V2 V0 V3 V2 V3</parameters>
</constraint>
<constraint name="C2" arity="3" scope="V0 V3 V4" reference="P0">
  <parameters>V0 V3 V0 V4 V3 V4</parameters>
</constraint>
<constraint name="C3" arity="3" scope="V0 V4 V5" reference="P0">
  <parameters>V0 V4 V0 V5 V4 V5</parameters>
</constraint>
<constraint name="C4" arity="3" scope="V0 V5 V6" reference="P0">
  <parameters>V0 V5 V0 V6 V5 V6</parameters>
</constraint>
<constraint name="C5" arity="3" scope="V0 V6 V7" reference="P0">
  <parameters>V0 V6 V0 V7 V6 V7</parameters>
</constraint>
<constraint name="C6" arity="3" scope="V0 V7 V8" reference="P0">
  <parameters>V0 V7 V0 V8 V7 V8</parameters>
</constraint>
<constraint name="C7" arity="3" scope="V0 V8 V9" reference="P0">
  <parameters>V0 V8 V0 V9 V8 V9</parameters>
</constraint>
<constraint name="C8" arity="3" scope="V0 V9 V10" reference="P0">
  <parameters>V0 V9 V0 V10 V9 V10</parameters>
</constraint>
<constraint name="C9" arity="3" scope="V0 V10 V11" reference="P0">
  <parameters>V0 V10 V0 V11 V10 V11</parameters>
</constraint>
<constraint name="C10" arity="3" scope="V1 V2 V4" reference="P0">
  <parameters>V1 V2 V1 V4 V2 V4</parameters>
</constraint>
<constraint name="C11" arity="3" scope="V1 V3 V5" reference="P0">
  <parameters>V1 V3 V1 V5 V3 V5</parameters>
</constraint>
<constraint name="C12" arity="3" scope="V1 V4 V6" reference="P0">
  <parameters>V1 V4 V1 V6 V4 V6</parameters>
</constraint>
<constraint name="C13" arity="3" scope="V1 V5 V7" reference="P0">
  <parameters>V1 V5 V1 V7 V5 V7</parameters>
</constraint>
<constraint name="C14" arity="3" scope="V1 V6 V8" reference="P0">
  <parameters>V1 V6 V1 V8 V6 V8</parameters>
</constraint>
<constraint name="C15" arity="3" scope="V1 V7 V9" reference="P0">
  <parameters>V1 V7 V1 V9 V7 V9</parameters>
</constraint>
<constraint name="C16" arity="3" scope="V1 V8 V10" reference="P0">
  <parameters>V1 V8 V1 V10 V8 V10</parameters>
</constraint>
<constraint name="C17" arity="3" scope="V1 V9 V11" reference="P0">
  <parameters>V1 V9 V1 V11 V9 V11</parameters>
</constraint>
<constraint name="C18" arity="3" scope="V2 V3 V6" reference="P0">
  <parameters>V2 V3 V2 V6 V3 V6</parameters>
</constraint>
<constraint name="C19" arity="3" scope="V2 V4 V7" reference="P0">
  <parameters>V2 V4 V2 V7 V4 V7</parameters>
</constraint>
<constraint name="C20" arity="3" scope="V2 V5 V8" reference="P0">
  <parameters>V2 V5 V2 V8 V5 V8</parameters>
</constraint>
<constraint name="C21" arity="3" scope="V2 V6 V9" reference="P0">
  <parameters>V2 V6 V2 V9 V6 V9</parameters>
</constraint>
<constraint name="C22" arity="3" scope="V2 V7 V10" reference="P0">
  <parameters>V2 V7 V2 V10 V7 V10</parameters>
</constraint>
<constraint name="C23" arity="3" scope="V2 V8 V11" reference="P0">
  <parameters>V2 V8 V2 V11 V8 V11</parameters>
</constraint>
<constraint name="C24" arity="3" scope="V3 V4 V8" reference="P0">
  <parameters>V3 V4 V3 V8 V4 V8</parameters>
</constraint>
<constraint name="C25" arity="3" scope="V3 V5 V9" reference="P0">
  <parameters>V3 V5 V3 V9 V5 V9</parameters>
</constraint>
<constraint name="C26" arity="3" scope="V3 V6 V10" reference="P0">
  <parameters>V3 V6 V3 V10 V6 V10</parameters>
</constraint>
<constraint name="C27" arity="3" scope="V3 V7 V11" reference="P0">
  <parameters>V3 V7 V3 V11 V7 V11</parameters>
</constraint>
<constraint name="C28" arity="3" scope="V4 V5 V10" reference="P0">
  <parameters>V4 V5 V4 V10 V5 V10</parameters>
</constraint>
<constraint name="C29" arity="3" scope="V4 V6 V11" reference="P0">
  <parameters>V4 V6 V4 V11 V6 V11</parameters>
</constraint>
</constraints>
</instance>
