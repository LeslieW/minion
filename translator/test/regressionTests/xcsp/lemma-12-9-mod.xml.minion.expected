MINION 3

#This file has been automatically generated by TAILOR v0.2
#bug-reports: andrea@cs.st-and.ac.uk

# amount of common subexpressions used:0
# amount of inferred common subexpressions used:0
# amount of original variables saved (over direct equality):0
# amount of constraints:90
# amount of auxiliary variables: 0

**VARIABLES**
DISCRETE V0 {0..8}
DISCRETE V1 {0..8}
DISCRETE V2 {0..8}
DISCRETE V3 {0..8}
DISCRETE V4 {0..8}
DISCRETE V5 {0..8}
DISCRETE V6 {0..8}
DISCRETE V7 {0..8}
DISCRETE V8 {0..8}
DISCRETE V9 {0..8}
DISCRETE V10 {0..8}
DISCRETE V11 {0..8}

# aliases

**SEARCH**

PRINT [
[V0],[V1],[V2],[V3],[V4],[V5],[V6],[V7],
[V8],[V9],[V10],[V11]]

VARORDER [
V0,V1,V2,V3,V4,V5,V6,V7,
V8,V9,V10,V11]


**CONSTRAINTS**

diseq(V6, V11)
diseq(V4, V11)
diseq(V4, V6)
diseq(V5, V10)
diseq(V4, V10)
diseq(V4, V5)
diseq(V7, V11)
diseq(V3, V11)
diseq(V3, V7)
diseq(V6, V10)
diseq(V3, V10)
diseq(V3, V6)
diseq(V5, V9)
diseq(V3, V9)
diseq(V3, V5)
diseq(V4, V8)
diseq(V3, V8)
diseq(V3, V4)
diseq(V8, V11)
diseq(V2, V11)
diseq(V2, V8)
diseq(V7, V10)
diseq(V2, V10)
diseq(V2, V7)
diseq(V6, V9)
diseq(V2, V9)
diseq(V2, V6)
diseq(V5, V8)
diseq(V2, V8)
diseq(V2, V5)
diseq(V4, V7)
diseq(V2, V7)
diseq(V2, V4)
diseq(V3, V6)
diseq(V2, V6)
diseq(V2, V3)
diseq(V9, V11)
diseq(V1, V11)
diseq(V1, V9)
diseq(V8, V10)
diseq(V1, V10)
diseq(V1, V8)
diseq(V7, V9)
diseq(V1, V9)
diseq(V1, V7)
diseq(V6, V8)
diseq(V1, V8)
diseq(V1, V6)
diseq(V5, V7)
diseq(V1, V7)
diseq(V1, V5)
diseq(V4, V6)
diseq(V1, V6)
diseq(V1, V4)
diseq(V3, V5)
diseq(V1, V5)
diseq(V1, V3)
diseq(V2, V4)
diseq(V1, V4)
diseq(V1, V2)
diseq(V10, V11)
diseq(V0, V11)
diseq(V0, V10)
diseq(V9, V10)
diseq(V0, V10)
diseq(V0, V9)
diseq(V8, V9)
diseq(V0, V9)
diseq(V0, V8)
diseq(V7, V8)
diseq(V0, V8)
diseq(V0, V7)
diseq(V6, V7)
diseq(V0, V7)
diseq(V0, V6)
diseq(V5, V6)
diseq(V0, V6)
diseq(V0, V5)
diseq(V4, V5)
diseq(V0, V5)
diseq(V0, V4)
diseq(V3, V4)
diseq(V0, V4)
diseq(V0, V3)
diseq(V2, V3)
diseq(V0, V3)
diseq(V0, V2)
diseq(V1, V2)
diseq(V0, V2)
diseq(V0, V1)

**EOF**
